fmriprep/interfaces/multiecho.py

```diff
@@ -12,25 +12,82 @@
 import os
 import numpy as np
 import nibabel as nb
+from nilearn.masking import (apply_mask, unmask)
 
 from niworkflows.nipype import logging
-from niworkflows.nipype.utils.filemanip import split_filename
+from niworkflows.nipype.utils.filemanip import (split_filename, fname_presuffix)
 from niworkflows.nipype.interfaces.base import (
     traits, TraitedSpec, File, InputMultiPath, SimpleInterface,
     BaseInterfaceInputSpec)
 
 LOGGER = logging.getLogger('interface')
 
 
+class FirstEchoInputSpec(BaseInterfaceInputSpec):
+    in_files = InputMultiPath(File(exists=True), mandatory=True,
```

Comment:
I'm not 100% this suggestion would work, but I'd love if you gave it a try:

```Python
in_files = InputMultiPath(File(exists=True), mandatory=True, minlen=2,
                          desc='multi-echo BOLD EPIs')
```

[In theory](https://github.com/enthought/traits/blob/080b722f72299acb63d539037107bd630f07ae35/traits/trait_types.py#L2292) List traits (as the InputMultiPath) allow for a `minlen` argument.